ui/src/qlcplus_fr_FR.ts

```diff
@@ -1342,34 +1338,34 @@ Celui-ci a dû être déplacé ou effacé.</translation>
         <location filename="chasereditor.ui" line="89"/>
         <location filename="chasereditor.cpp" line="979"/>
         <source>Fade In</source>
-        <translation>Montée</translation>
+        <translation>Fondu entrant</translation>
     </message>
     <message>
         <location filename="chasereditor.ui" line="99"/>
         <location filename="chasereditor.cpp" line="980"/>
         <source>Fade Out</source>
-        <translation>Descente</translation>
+        <translation>Fondu sortant</translation>
     </message>
     <message>
         <location filename="chasereditor.ui" line="104"/>
         <source>Duration</source>
-        <translation>Durée du pas</translation>
+        <translation>Durée</translation>
     </message>
     <message>
         <location filename="chasereditor.ui" line="741"/>
         <source>Add step(s) to the current position</source>
-        <translation>Ajouter un ou plusieurs pas à la position actuelle</translation>
+        <translation>Ajouter une(des) étape(s) à la position actuelle</translation>
     </message>
     <message>
         <location filename="chasereditor.ui" line="595"/>
         <source>Show/Hide speed dial window</source>
-        <translation>Afficher/Masquer la fenêtre de réglage des vitesses</translation>
+        <translation>Afficher/Masquer la fenêtre molette rapide</translation>
     </message>
     <message>
         <location filename="chasereditor.ui" line="94"/>
         <location filename="chasereditor.cpp" line="981"/>
         <source>Hold</source>
-        <translation>Maintien</translation>
+        <translation>Maintenir</translation>
```

Comment:
ici "hold" c'est la partie du milieu de l'enchainement

fade in -> hold -> fade out

Ca devrait rester un nom.

(Et de ce que je me souviens "duration" c'est fadein+hold, c'est pour ça que à la fois duration et hold existent)